.docker/app/Dockerfile

```diff
@@ -1,12 +1,24 @@
-FROM ruby:2.6
-RUN \
-      wget -qO- https://deb.nodesource.com/setup_9.x | bash - && \
-      wget -qO- https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
-      echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
-      apt-get update -qq && \
-      apt-get install -y --force-yes --no-install-recommends && \
-      nodejs yarn build-essential libpq-dev \
-      && apt-get clean
-RUN gem install bundler
+FROM ruby:2.6.5-alpine
```

Comment:
`docker` noob ✋  

I've got a couple questions because I'm just trying to learn.

So why 2.6.5-alpine? Specifically the `alpine` part. There are [a lot of flavors to choose from on docker hub](https://hub.docker.com/_/ruby). Also, why 2.6.5 and not some other version? 2.7 is the current stable release and Blacklight's docs say it can be run on ruby 2.2+.

Also wondering if this could be set to respect `.ruby-version` or is that missing the point?

Reply:
alpine is the better choice ... we're trying to switch to that at TU too.  It's because alpine images are smaller and they are more secure.. .so using them in production is smarter.

Reply:
Guess using them everywhere is smarter too then, right? Or should I not make that assumption. Thanks.

Reply:
I don't believe there is a way to read the `.ruby-version` file, and I don't think Blacklight ships with a `.ruby-version` file.

I'll push a change momentarily that allows overriding the version of the Alpine Ruby base image via environment variable, defaulting to 2.6.5 (because 2.6 is what was in here previously, and 2.6.5 is the specific 2.6 version we are currently testing, but it's arbitrary). Sufficient?

Reply:
@cdmo :speech_balloon: 
> Guess using them everywhere is smarter too then, right? Or should I not make that assumption. Thanks.

Yeah, we use Alpine-based images for dev/test generally as well because they're lighter.